sql/moz-fx-data-shared-prod/monitoring_derived/outerbounds_cost_per_flow_run_v1/view.sql

```diff
@@ -0,0 +1,21 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.monitoring_derived.outerbounds_cost_per_flow_run_v1`
+AS
+SELECT
+  DATE_TRUNC(usage_start_time, DAY) AS invoice_day,
+  labels.value AS run_id,
+  IFNULL(SAFE.LEFT(labels.value, INSTR(labels.value, '-', -1) - 1), labels.value) AS flow_name,
+  SUM(cost) AS cost_usd
```

Comment:
@chelseatroy We missed `credits.amount` in total cost (confirmed by @jasonthomas) like this:

```
SUM(cost) + SUM(IFNULL((SELECT SUM(c.amount)  FROM  UNNEST(credits) c), 0)) AS cost_usd
```
as per https://cloud.google.com/billing/docs/how-to/bq-examples#costs-details-per-invoice.